### ISSUE TITLE
Request.clone/Response.clone do not specify in which global the new object belongs to.

### ISSUE THREAD

**Issue by bakulf** [whatwg/fetch#825]
In which global should be created the new request? Same question for a cloned Response object.
Currently, in Firefox we use the global of the original request/response object.

**Comment by annevk** [comment]
Yeah me too, that way any prototype expandos also remain available etc. I'll see about testing this at least. I think the behavior would fall out of fixing https://github.com/heycam/webidl/issues/135 in IDL.
